lib/classes/squad.dart

```diff
@@ -0,0 +1,93 @@
+import 'package:collection/collection.dart';
+import 'package:phone_factory/classes/squad_members/squad_member.dart';
+
+class Squad {
+  int _captain = 1;
+  static const int _initSquadMemberCount = 3;
+  List<SquadMember> _squad = [];
+  int _antidotCount = 1;
+  List<SquadMember> get squad => _squad;
```

Comment:
you have setter and getter, so you are incapsulate nothing. remove incapsulation.